cider-velox/src/ciderTransformer/CiderPlanTransformerOptions.h

```diff
@@ -26,3 +26,5 @@ DECLARE_bool(left_deep_join_pattern);
 DECLARE_bool(compound_pattern);
 DECLARE_bool(filter_pattern);
 DECLARE_bool(partial_agg_pattern);
+DECLARE_bool(top_n_pattern);
+DECLARE_bool(order_by_pattern);
```

Comment:
Hmm, how is this bool variable used? It seems we always register them in https://github.com/intel/BDTK/pull/179/files#diff-4d0a60b50d3aa571f2bec5fd249b2d20fc5b3b7a2a8cb75c2789a6be043c07e4R31